src/components/line-chip/line-chip.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+  min-width: 4.125rem;
```

Comment:
<img width="102" alt="2023-11-10 at 14 13 37@2x" src="https://github.com/AtB-AS/planner-web/assets/14045515/783d36ce-e477-47ab-8ccc-92a51d3cb700">


The line chip looks a bit off for me with single digit public codes.

Is this `min-width` needed?

Reply:
This was done to ensure the same width for all chips in scenarios like this: 

<img width="425" alt="image" src="https://github.com/AtB-AS/planner-web/assets/43166974/82b2194f-aac8-4a47-be06-b615bf9fca17">

We might want to rethink this.

Reply:
Right, of course. Let's keep it as is.